src/ray/util/compat.cc

```diff
@@ -97,6 +98,7 @@ Status Close(MEMFD_TYPE_NON_UNIQUE fd) {
   }
   return Status::OK();
 }
+int64_t GetPid() { return (int64_t)(GetCurrentProcessId()); }
```

Comment:
getpid() exists on windows

```
#ifdef _WIN32
#include <process.h>  // to ensure getpid() on Windows
#endif
```

from util.h

Reply:
It's not called: https://github.com/ray-project/ray/blob/765922126c96e6adce838c5e3f278c7516765e26/src/ray/util/process.cc#L636-L642
But I found we already have it in process.h

Reply:
I see. we should unify them.